target/linux/rtl838x/image/Makefile

```diff
@@ -0,0 +1,44 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+include $(INCLUDE_DIR)/image.mk
+
+KERNEL_LOADADDR = 0x80000000
+KERNEL_ENTRY = 0x80000400
+
+define Device/Default
+  PROFILES = Default
+  KERNEL := kernel-bin | append-dtb | gzip | uImage gzip
+  KERNEL_INITRAMFS := kernel-bin | append-dtb | gzip | uImage gzip
+  DEVICE_DTS_DIR := ../dts
+  DEVICE_DTS = $$(SOC)_$(1)
+  IMAGES := sysupgrade.bin
+  SUPPORTED_DEVICES := $(subst _,$(comma),$(1))
+  sysupgrade_bin := append-kernel | pad-to 64k | append-rootfs | pad-rootfs
```

Comment:
I'm not a fan of these "shortcut" variables, and since you don't actually need it (yet), I'd prefer to remove it.

Reply:
Fixed.